### DISCUSSION TITLE
Vitest process exited with code 0

### DISCUSSION THREAD

**Discussion by mil7** [vitest-dev/vscode#378] [Q&A]
Hi there,

Since the latest versions the vitest extension is not working anymore in my VSC. The error output seems to be very vague. Too vague to really submit a bug. I believe it is caused by the big changes in the plugin in the last weeks. I did not use the extension for at least two months.

I can reproduce this with minimum efford by setting up a base vite application with `npm create vite@latest my-vue-app -- --template vue` enhanced with vitest.

I am using a windows machine with Node v20.12.2 and npm v10.5.0.

```
[INFO 4:24:53 PM] [v0.10.1] Vitest extension is activated because Vitest is installed or there is a Vite/Vitest config file in the workspace.
[INFO 4:24:53 PM] [API] Running Vitest: v1.6.0 (vite.config.ts)
[INFO 4:24:53 PM] [API] Starting Vitest process with Node.js: C:\Program Files\Volta\node.EXE
[Error 4:24:53 PM] [Error Error] Vitest process exited with code 0
Error: Vitest process exited with code 0
	at ChildProcess.d (c:\Users\xxx\.vscode\extensions\vitest.explorer-0.10.1\dist\extension.js:21:170)
	at ChildProcess.emit (node:events:517:28)
	at ChildProcess._handle.onexit (node:internal/child_process:292:12)
```

Any ideas?

**Comment by programmer24601** [comment]
I'm getting a similar error on Ubuntu 22.04.4 LTS, Node 20.11.1, npm 10.2.4, vitest 1.5.3, vitest-vscode extension 10.0.1.

**The latest version of the vitest extension to fully work for me is `0.4.1`.**


Experimenting with right-click -> Install Another Version...
- `0.6.0 - 0.8.5` - no "Output" errors, no test run buttons, and the Testing pane contains: Vitest -> Resolving Vitest...
- `0.8.6 - 0.10.1` - the below "Output" error, no test run buttons, and Testing pane is empty.

```
[INFO 20:04:51] [v0.10.1] Vitest extension is activated because Vitest is installed or there is a Vite/Vitest config file in the workspace.
[INFO 20:04:51] [API] Running Vitest: v1.5.3 (vitest.workspace.d.ts), v1.5.3 (vitest.workspace.js), v1.5.3 (vitest.workspace.ts)
[INFO 20:04:51] [API] Starting Vitest process with Node.js: /home/me/.asdf/shims/node
[Worker] ReferenceError: _default is not defined
[Error 20:04:52] [Error Error] Vitest process exited with code 1
Error: Vitest process exited with code 1
	at ChildProcess.d (/home/me/.vscode/extensions/vitest.explorer-0.10.1/dist/extension.js:21:170)
	at ChildProcess.emit (node:events:517:28)
	at ChildProcess._handle.onexit (node:internal/child_process:292:12)
```

**Reply by mil7** [reply]
A bug report has been published making a correlation to volta: https://github.com/vitest-dev/vscode/issues/382
